workflow/scripts/modify_prenetwork.py

```diff
@@ -332,6 +332,14 @@ def transmission_costs_from_modified_cost_data(n, costs, transmission, length_fa
     )
     n.links.loc[dc_b, "capital_cost"] = costs
 
+def must_run_biomass(n, p_min_pu, regions):
+    """
+    Set p_min_pu for biomass generators to the specified value.
+    """
+    logger.info(f"Setting p_min_pu = {p_min_pu} for biomass generators")
```

Comment:
For more clarity i would suggest:

    logger.info(f"Must-run condition enabled: Setting p_min_pu = {p_min_pu} for biomass generators.")